tests/spec/core/biblio-spec.js

```diff
@@ -14,37 +15,147 @@ describe("W3C — Bibliographic References", () => {
     previousPublishDate: "2013-12-17",
     localBiblio: {
       Zzz: {
-        title: "Last Reference",
+        title: "Last Reference"
       },
       aaa: {
-        title: "First Reference",
+        title: "First Reference"
       },
       TestRef1: {
         title: "Test ref title",
         href: "http://test.com",
         authors: ["William Shakespeare"],
-        publisher: "Publishers Inc.",
+        publisher: "Publishers Inc."
       },
       TestRef2: {
         title: "Second test",
         href: "http://test.com",
         authors: ["Another author"],
-        publisher: "Testing 123",
+        publisher: "Testing 123"
       },
       TestRef3: {
         title: "Third test",
         href: "http://test.com",
-        publisher: "Publisher Here",
+        publisher: "Publisher Here"
       },
       FOOBARGLOP: {
-        aliasOf: "BARBAR",
+        aliasOf: "BARBAR"
       },
       BARBAR: {
-        title: "The BARBAR Spec",
+        title: "The BARBAR Spec"
+      }
+    }
+  };
+
+  //Configuration for APA citation
+  const conf2 = {
```

Comment:
here, you can just do:

```JS
const confAPA = {
   ...confDefault,
   biblioStyle: "APA",
};

const confMLA = {
   ...confDefault,
   biblioStyle: "MLA",
};
```

So then you don't need all the duplicate code ✨

Reply:
Spread the values, got it.